src/schema/users.ts

```diff
@@ -491,7 +501,26 @@ export const resolvers: IResolvers<any, Context> = {
           : null,
       };
     },
-    user: (_, { id }: { id: string }): Promise<GQLUser> => fetchUserById(id),
+    user: async (
+      _,
+      { id }: { id: string },
+      ctx: Context,
+      info: GraphQLResolveInfo,
+    ): Promise<GQLUser> => {
+      const res = await graphorm.query<GQLUser>(ctx, info, (builder) => {
+        builder.queryBuilder = builder.queryBuilder
+          .andWhere(
+            `("${builder.alias}"."id" = :id OR "${builder.alias}"."username" = :id)`,
+            { id },
+          )
+          .limit(1);
+        return builder;
+      });
+      if (!res[0]) {
+        throw new ForbiddenError('user not found');
```

Comment:
Do we throw Forbidden or not found usually?

Reply:
We changed it, this is the general user call.
In the graphql adapter we made it forbidden error for the whoami call.

Reply:
In the whoami it makes sense because you are not logged in so you should receive forbidden if there's an issue.
But here the user doesn't exist, so it's better I think to throw NotFound